### ISSUE TITLE
Hierarchical Kinematic Probability Distributions for 3D Human Shape and Pose Estimation from Images in the Wild

### ISSUE THREAD

**Issue by Arata2887** [Arata2887/issue_copy#31]
## 0.どんな論文? /What does this work do?

- https://arxiv.org/pdf/2110.00990v2
- ICCV 2021
- Introduces train a deep neural network to estimate a hierarchical matrix-Fisher distribution over relative 3D joint rotation matrices (i.e. body pose), which exploits the human body’s kinematic tree structure, as well as a Gaussian distribution over SMPL body shape parameters.

## 1. 論文のモチベーション / Motivation(s) of the paper / document

> <img width="550" alt="Fig 1" src="https://github.com/Arata2887/issue_copy/assets/119403000/edfe9f07-8450-44dd-a708-a789bf15f75d">

- In 3D human body shape and pose estimation, multiple plausible 3D configurations can explain a single 2D image, especially when parts of the body are occluded. 
- Traditional methods often predict a single deterministic solution, which fails to capture this uncertainty. 
- This paper aims to improve the robustness and accuracy of 3D reconstructions by predicting a distribution over possible shapes and poses, thereby better handling occlusions and ambiguities.

### Hierarchical Probabilistic 3DHuman

> <img width="859" alt="Fig 2" src="https://github.com/Arata2887/issue_copy/assets/119403000/ec7d234e-efdd-4db8-b036-6ca39b330988">

**Input Representation:**
- The input to the model is an edge-image concatenated with joint heatmaps, rather than a single RGB image.
- This representation helps bridge the domain gap between synthetic training data and real-world images.

**Hierarchical Matrix-Fisher Distribution**
- The method predicts a hierarchical matrix-Fisher distribution over 3D joint rotations.
- This distribution is conditioned on the kinematic tree structure of the human body.
- It takes into account the dependencies between joint rotations in the kinematic hierarchy.

**Gaussian Distribution for Body Shape**
- A Gaussian distribution is used for the SMPL body shape parameters.
- The **mean** and **variance** of the distribution are predicted.
- This allows for the modeling of uncertainty in the shape estimation.

**Differentiable Rejection Sampler:**
- To ensure the predicted distributions match the 2D visual evidence in the input image, a differentiable rejection sampler is used.
- This sampler imposes a reprojection loss between ground-truth 2D joint coordinates and samples from the predicted distributions, which enables the backpropagation of gradients through the sampling process, ensuring consistency between the 3D predictions and the 2D input.

### Quantitative results

**Table 2** show the comparison of methods on 3DPW shows the proposed method with HRNet-W48 achieves top pose estimation accuracy.

> <img width="385" alt="Table 2" src="https://github.com/Arata2887/issue_copy/assets/119403000/91a18b4d-c8d2-40c0-9d18-26703f0e8bc1">

**Table 3** show the evaluation on SSP-3D demonstrates the proposed method excels in both single-image and multi-image scenarios.

> <img width="445" alt="Table 3" src="https://github.com/Arata2887/issue_copy/assets/119403000/60bf2c2d-e41d-4713-9a3d-d671fe643c7d">

**Figure 4** show the Sorted per-sample distributions on 3DPW and SSP-3D highlight the superior performance of the proposed method.

> <img width="410" alt="Fig 4" src="https://github.com/Arata2887/issue_copy/assets/119403000/303056c3-cdf5-4de0-ac91-991c9aa1148b">

### Qualitative results
Although there is no direct comparison with other methods, **8. Qualitative Results** show some qualitative output of the proposed method, as detailed in the paper.

> <img width="577" alt="Fig 6" src="https://github.com/Arata2887/issue_copy/assets/119403000/625aa5ec-2466-431f-ae08-5ce4381c8ab0">
> <img width="577" alt="Fig 7" src="https://github.com/Arata2887/issue_copy/assets/119403000/a8699d92-9bf9-4274-8155-5925c9ea28e6">
> <img width="581" alt="Fig 8" src="https://github.com/Arata2887/issue_copy/assets/119403000/b5ae5a19-674e-4625-8627-2176dfa2559c">

### Ablation studies
**Table 1** shows ablation study results, indicating that the hierarchical architecture with edge + joint heatmaps input and 2D samples loss achieves the best performance across metrics.

> <img width="850" alt="Table 1" src="https://github.com/Arata2887/issue_copy/assets/119403000/8d041e79-abd5-4216-8755-46c118787e6b">

## 4.公開コードやデータセットは? /Public codes or datasets?

- https://github.com/akashsengupta1997/HierarchicalProbabilistic3DHuman
